crates/network-scanner-net/Cargo.toml

```diff
@@ -9,10 +9,12 @@ publish = false
 [dependencies]
 anyhow = "1.0.79"
 crossbeam = { version = "0.8.4", features = ["crossbeam-channel"] }
+futures = "0.3.30"
 parking_lot = "0.12.1"
-polling = "3.3.2"
+polling = {git = "https://github.com/irvingoujAtDevolution/polling.git"}
```

Comment:
a updated version to polling, allow us to kown weather tcp connection has failed or not.

Reply:
waiting to be merged into polling main branch https://github.com/smol-rs/polling/pull/185

Reply:
It is merged, I will update it once they publish a new version